docker/Dockerfile

```
@@ -0,0 +1,42 @@
+# TODO: update version
+FROM rocker/r-ver:4.4.1
+
+RUN apt-get update &&  apt-get install -y --no-install-recommends \
+        libcurl4-openssl-dev \
+        libssl-dev \
+        libv8-dev \
+        zlib1g-dev \
+        && apt-get clean \
+        && rm -rf /var/lib/apt/lists/*
+
+# NOTE: manual install stringi to link against libicu70 over libicu66
+RUN install2.r --error stringi
+
+# Without this, we are unable to pick up more recent packages
+COPY docker/Rprofile.site /usr/local/lib/R/etc/Rprofile.site
+
+# More cache-friendly installation of some dependencies:
+# NOTE: need to convert to using r-universe instead
+RUN install2.r --error \
+        --repos=https://mrc-ide.github.io/drat \
```

Can you update this

```suggestion
        --repos=https://mrc-ide.r-universe.dev \
```

everything should just work with that in place